Update GCP connect doc to include more actionable instructions for client (#3431)
astronomer/docs: astro/connect-gcp.md

Astro clusters are by default configured with a PSC endpoint with a target of [All Google APIs](https://cloud.google.com/vpc/docs/configure-private-service-connect-apis#supported-apis). To provide a secure-by-default configuration, a DNS zone is created with a resource record that will route all requests made to `*.googleapis.com` through this PSC endpoint. This ensures that requests made to these services are made over PSC without any additional user configuration. As an example, requests to `storage.googleapis.com` will be routed through this PSC endpoint.

> You can check if the service to which you’re attempting to connect Airflows is available through the All Google APIs target by running: 

Reviewer: ```suggestion
You can check if the service that you want to connect Airflow to is available through the **All Google APIs** target by running the following command: 
```

```bash
gcloud services list --available --filter="name:googleapis.com"
```

> If you do NOT see your service listed, please open a support case with Astronomer support to set up the necessary PSC connectivity. 

Reviewer: ```suggestion
If you don't see your service listed, please open a support case with [Astronomer support](astro-support.md) to set up the necessary PSC connectivity. 
```

> If you don't see your service listed, please open a support case with [Astronomer support](astro-support.md) to set up the necessary PSC connectivity. 

Reviewer: ```suggestion
If you don't see your service listed, open a support case with [Astronomer support](astro-support.md) to set up the necessary PSC connectivity. 
```
